src/GafferSceneTestModule/GafferSceneTestModule.cpp

```diff
@@ -67,7 +67,9 @@ BOOST_PYTHON_MODULE( _GafferSceneTest )
 
 	GafferBindings::DependencyNodeClass<CompoundObjectSource>();
 	GafferBindings::NodeClass<TestShader>();
-	GafferBindings::NodeClass<TestLight>();
+	GafferBindings::NodeClass<TestLight>()
+		.def( "loadShader", (void (TestLight::*)(const std::string & ) )&TestLight::loadShader )
```

Comment:
Do we actually need the `(void (TestLight::*)...` cast here?

Reply:
Yes, looks like it works fine without the cast. Fixed in https://github.com/GafferHQ/gaffer/pull/5846/commits/5322043c418794f7cdff1eb6fb7dadceb985f1e1